### ISSUE TITLE
[4.0.0] Tree support uses a lower value of Support Bottom Distance

### ISSUE THREAD

**Issue by mbartlett21** [Ultimaker/Cura#5807]
**Application Version**
4.0.0

**Platform**
Windows 10

**Steps to Reproduce**
Add below model to cura.
Disable support and enable tree support. with default settings.
Set support bottom distance to 0.1.

**Actual Results**
Tree support starts immediately on the circled face

**Expected results**
Tree support would allow a layer between the circled face and the start of the tree support, like standard support does.

Not until the support bottom distance is increased beyond the layer height does the tree support allow a layer betrween.

**Additional Information**
![image](https://user-images.githubusercontent.com/29034492/58560466-8ae53e80-8268-11e9-980e-08b8e41f79be.png)

Thingiverse STL: https://www.thingiverse.com/download:2364489

**Comment by mbartlett21** [comment]
Sorry,

Duplicate of https://github.com/Ultimaker/Cura/issues/5759
